src/users/dto/update-user.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { CreateUserDto } from './create-user.dto';
+import { PartialType } from '@nestjs/mapped-types';
+
+export class UpdateUserDto extends PartialType(CreateUserDto) {}
```

Comment:
**praise**: Extending `PartialType` from `CreateUserDto` is a clean and efficient way to handle optional fields in updates.